extensions/rich_text_components/Collapsible/Collapsible.html

```diff
@@ -1,4 +1,10 @@
 <script type="text/ng-template" id="richTextComponent/Collapsible">
+  <style>
+    .conversation-skin-tutor-card-top-content.protractor-test-conversation-content {
```

Comment:
Thanks for the screenshots @luskjh, they were very helpful!

And yes, I think it is okay for all accordions to be the full width of the card. I think that's more natural behaviour, in any case.

(/cc @amylatten, @anuzis for info -- I think this has come up in our conversations before, too.)